ansible/azure/start_migration.yaml

```diff
@@ -33,7 +33,7 @@
   - name: <== Executing payload ==>
     shell: "python3 /tmp/start.py {{ url }} '{{ sas }}' {{ mongodb }} {{ project }} {{ hostname }}"
     register: st
-    become: True
+    become: true
```

Comment:
This become is redundant because become is set to true for the whole play on the top